Added README section for setting fstl as default viewer

nnadeau opened the pull request:
## Changed

- Minor markdown whitespace fixes
- Added README section for setting fstl as default viewer

## Context

- Related to #40

mkeeter commented:
Can you add which OSes this works for?

sur5r commented:
`mimeopen` is part of the `File::MimeInfo` perl module. It writes to `$XDG_CONFIG_HOME/mimeapps.list`.

As `mimeopen` is not usually installed, it would be best to use `xdg-mime default ...` instead. This, however would require to ship a `.desktop` file with `fstl`, which in turn would also enable to set these defaults using whatever GUI tool the desktop environment ships.

nnadeau commented:
@mkeeter, added windows + macos sections and labelled sections according to OS

@sur5r , completely agree that it depends on a not-always-available (albeit quite common) package. I updated the section to reference the package's availability.
